create-project.py

```diff
@@ -45,6 +65,20 @@
     else:
         exit()
 
+zip_destination, _ = urllib.request.urlretrieve(files_url, current_dir / "relm4-template.zip")
```

Comment:
Can we make this optional? I think for local testing it would be nice not to download the zip file.

Reply:
How about something like this?

```suggestion
if len(sys.argv) > 0 and sys.argv[0] == "--online":
    zip_destination, _ = urllib.request.urlretrieve(files_url, current_dir / "relm4-template.zip")

    with zipfile.ZipFile(zip_destination, 'r') as zip_ref:
        zip_ref.extractall(current_dir)

    os.remove(zip_destination)

    for item in os.listdir(template_path):
        item_path = os.path.join(template_path, item)
        if "create-project.py" not in item_path:
            remove_path = os.path.join(current_dir, item)
            shutil.move(item_path, remove_path)

    shutil.rmtree(template_path)
```

Reply:
Looks good, but we should document it I think

Reply:
README?

Reply:
yes

Reply:
`README.md` updated.